staging/src/slime.io/slime/modules/limiter/controllers/envoy_filters.go

```diff
@@ -143,10 +156,15 @@ func descriptorsToEnvoyFilter(descriptors []*microservicev1alpha2.SmartLimitDesc
 
 	// enable and config plugin envoy.filters.http.local_ratelimit
 	if len(localDescriptors) > 0 {
-		httpFilterLocalRateLimitPatch := generateHttpFilterLocalRateLimitPatch()
-		ef.ConfigPatches = append(ef.ConfigPatches, httpFilterLocalRateLimitPatch)
-
-		perFilterPatch := generateLocalRateLimitPerFilterPatch(localDescriptors, loc)
+		// TODO if disableInsertLocalRateLimit=false in gw, multi smartlimiter will patch duplicate configurations, This will lead dysfunctional behavior.
```

Comment:
Seems nothing todo, or you want to improve this solution in the future? Then intro the idea more clearly.